ab_tool/exceptions.py

```diff
@@ -26,6 +26,16 @@
 TRACK_WEIGHTS_NOT_SET = Renderable404("Missing track configuration. If you are a student, notify your course instructors regarding this page before returning here.")
 INPUT_NOT_ALLOWED = Renderable404("Input does not follow requirements")
 
+MISSING_NAME_PARAM = Renderable400("Names are required. Please go back and submit a name for all experiments, tracks, and intervention points.")
+INCORRECT_WEIGHTING_PARAM = Renderable400("Weights (between 0 and 100) are required if you are randomizing with weights. Please go back and submit weights for all tracks.")
+INVALID_URL_PARAM = Renderable400("One of your URL inputs is invalid.")
+
+PARAM_LENGTH_EXCEEDS_LIMIT = Renderable400("There are length limitations. At least one of your inputs is too long.")
+
+
+def DATABASE_ERROR(error_message):
```

Comment:
Can we do without this method and simply raise a `Renderable400` in the one place this is being used?